fix RTD documentation build for non-master branches

behackl opened the pull request:
## Explanation for Changes
We have been installing manim twice during the setup of the RTD build environment; the installation of the master branch overwrote the installation of the checked out changes leading to all branches building the documentation of the current master branch.

This is fixed here by removing `https://github.com/ManimCommunity/manim/archive/master.zip` from the requirements (the local `manim(lib)` package is installed nonetheless due to the configuration in `.readthedocs.yml`).

## Testing Status
Lets see what the pipeline does with this.

## Acknowledgement
- [x] I have read the [Contributing Guidelines](https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP))

behackl commented:
The changes in the documentation of this branch do show up again, so this actually fixes the issue:

![image](https://user-images.githubusercontent.com/11851593/93528578-d37faa00-f93a-11ea-9c84-0b0ba9e153a5.png)
(via https://manimce--449.org.readthedocs.build/en/449/reference/manim.mobject.geometry.html#module-manim.mobject.geometry)

Review at will!

naveen521kk reviewed (approved):
I just added that zip file because previous there was git install hard-coded so I changed it. I never know RTD installs the checked out branch.

behackl commented:
> I just added that zip file because previous there was git install hard-coded so I changed it. I never know RTD installs the checked out branch.

It only does since I introduced the `.readthedocs.yml`, removing the other install was probably just an oversight.

behackl commented:
It irritates me that RTD builds correctly on the other merge requests in this repository, but did not do so in mine (I've deleted my RTD instance, so can't link to it anymore).

Anyhow, the double installation still happens -- could someone else have a quick look at this so that we can merge this fix?

leotrs commented:
My bad - I didn't see that @naveen521kk had already accepted.
